src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,27 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.List;
+
+
+/**
+ * Lists all persons in the address book to the user.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Sort all persons in the address book.\n"
+            + "Example: " + COMMAND_WORD;
+
+    public static final String MESSAGE_SUCCESS = "Persons sorted";
+
+
```

Comment:
Stray line break? It is a good practice to check the diff of the PR and fix any problems before submitting it.